### PR TITLE
Update readme about vs

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md [hunk 1]
```diff
@@ -401,3 +401,26 @@ To connect to the docker on SSL, both the docker image and your local machine mu
 In the future, passing in a custom cert via a bound volume that contains your certificate will be a possibility as well.
 
 For additional reading on this process for trusting SSL certs locally, feel free to read up [here.](https://devblogs.microsoft.com/aspnet/configuring-https-in-asp-net-core-across-different-platforms/) The afore-mentioned [docker specific readme](../docker/README.md) also has details that are relevant.
+
+## Troubleshooting
+
+### Visual studio
+
+If you get the message dialog `The project doesn't know how to rin the profile Azure.Sdk.Tools.TestProxy`, you can fix it by reviewing the next two things:
```
Comment (benbp) [inline tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:409]:
```suggestion
If you get the message dialog `The project doesn't know how to run the profile Azure.Sdk.Tools.TestProxy`, you can fix it by reviewing the next two things:
```

Also, are these next two things the below ASP.NET and Windows IIS headers? If so I think they should be a lower level of nesting than the current one (`####` vs. `###`).

File: tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md [hunk 2]
```diff
@@ -401,3 +401,26 @@ To connect to the docker on SSL, both the docker image and your local machine mu
...
+
+### ASP.NET and web development
+
+Run Visual Studio installer and make sure ASP.NET and web development is installed.
+
+![image](https://user-images.githubusercontent.com/24213737/152257876-be1ed946-20bc-47ff-83da-f9ae05db290a.png)
+
+Then, look in the right panel that `Development tine IIS support` is not checked:
```
Comment (benbp) [inline tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:417]:
```suggestion
Then, confirm in the right panel that `Development time IIS support` is not checked:
```

File: tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md [hunk 3]
```diff
@@ -401,3 +401,26 @@ To connect to the docker on SSL, both the docker image and your local machine mu
...
+
+![image](https://user-images.githubusercontent.com/24213737/152257948-c61e6876-eb36-4414-b8de-8c85aa0532bb.png)
+
+### Windows IIS
+
+[Add Internet Information](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-6.0) Services to your Windows instalation. Here the list of features to enable:
```
Comment (benbp) [inline tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:423]:
```
[Add Internet Information](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/development-time-iis-support?view=aspnetcore-6.0) Services to your Windows installation. Here is the list of features to enable:
```

File: tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md [hunk 4]
```diff
@@ -401,3 +401,26 @@ To connect to the docker on SSL, both the docker image and your local machine mu
...
 
 For additional reading on this process for trusting SSL certs locally, feel free to read up [here.](https://devblogs.microsoft.com/aspnet/configuring-https-in-asp-net-core-across-different-platforms/) The afore-mentioned [docker specific readme](../docker/README.md) also has details that are relevant.
+
+## Troubleshooting
+
+### Visual studio
+
+If you get the message dialog `The project doesn't know how to rin the profile Azure.Sdk.Tools.TestProxy`, you can fix it by reviewing the next two things:
+
+### ASP.NET and web development
```
Comment (vhvb1989) [inline tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:411]:
```suggestion
#### ASP.NET and web development
```

File: tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md [hunk 5]
```diff
@@ -401,3 +401,26 @@ To connect to the docker on SSL, both the docker image and your local machine mu
...
+
+Run Visual Studio installer and make sure ASP.NET and web development is installed.
+
+![image](https://user-images.githubusercontent.com/24213737/152257876-be1ed946-20bc-47ff-83da-f9ae05db290a.png)
+
+Then, look in the right panel that `Development tine IIS support` is not checked:
+
+![image](https://user-images.githubusercontent.com/24213737/152257948-c61e6876-eb36-4414-b8de-8c85aa0532bb.png)
+
+### Windows IIS
```
Comment (vhvb1989) [inline tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md:421]:
```suggestion
#### Windows IIS
```

**Review by benbp — APPROVED** [review]

**Pull request merged**